rllib/algorithms/algorithm_config.py

```diff
@@ -2184,6 +2192,13 @@ def multi_agent(
             for pid in policies:
                 validate_policy_id(pid, error=True)
 
+            if self._enable_rl_module_api:
+                raise ValueError(
+                    "`policies` uses the Policy and ModelV2 API. This is incompatible "
+                    "with the RLModule API. Please set `_enable_rl_module_api=False` "
+                    "to use the legacy Policy and ModelV2 API."
+                )
+
```

Comment:
This is not compatible anymore, while policies_to_train and policy_mapping_fn can sitll be used.